.github/workflows/plugin-ci.yml

```diff
@@ -100,7 +107,7 @@ jobs:
         uses: mattermost/actions/plugin-ci/build@cd879ea9c64cc3e26a75a042d1c5066be28130a6
 
   delivery:
-    if: ${{ github.repository_owner == 'mattermost' && github.event_name != 'schedule' && (github.ref_name  == 'master' || github.ref_name  == 'main') }}
+    if: ${{ github.repository_owner == 'mattermost' && github.event_name != 'schedule' && (github.ref_name  == 'master' || github.ref_name  == 'main') && !inputs.skip-aws-upload }}
```

Comment:
I will have to test that . I do not think this is evaluated as boolean . 
Check [here](https://github.com/actions/runner/issues/1483)

Reply:
Oh, I forgot about that, but you're right, we need to use `fromJSON` or use string comparison.